pilot/docker/Dockerfile.proxyv2

```diff
@@ -45,5 +134,39 @@ COPY stats-filter.compiled.wasm /etc/istio/extensions/stats-filter.compiled.wasm
 COPY metadata-exchange-filter.wasm /etc/istio/extensions/metadata-exchange-filter.wasm
 COPY metadata-exchange-filter.compiled.wasm /etc/istio/extensions/metadata-exchange-filter.compiled.wasm
 
+ARG SDK_VERSION="2.15.100.3"
+ARG DCAP_VERSION="1.12.100.3"
```

Comment:
SGX SDK and DCAP versions are different from the ones used in the build [above](https://github.com/intel/istio/pull/3/files#diff-3696302fd9b6db61d2c5b15bc3d35575ee7e35cf758aab7ef8a9e7e5b140e2beR10-R12) layer.